tools/importers/helpers/typographyVariables.ts

```diff
@@ -0,0 +1,53 @@
+const typographyPrefix: string = 'typography-';
+const tokensKey: string = 'TypographyTokens';
+const excludedKeys: Array<string> = ['fontFamilies', 'TypographyTokens'];
+const fontFamilyProperty: string = 'font-families';
+const fontWeightKey: string = 'fontWeight';
+const transformCssProperty: { [key: string]: string } = {
+	'text-case': 'text-transform',
+	'-regular': '-normal',
+};
+const importVariables: string = "@import 'variables';";
+const jsonAttributeToCssProperty: { [key: string]: string } = {
+	lineHeight: 'line-height',
+	fontWeight: 'font-weight',
+	fontSize: 'font-size',
+	letterSpacing: 'letter-spacing',
+	textCase: 'text-transform',
+	textDecoration: 'text-decoration',
+};
+const resultCssAdditionalLines: Array<string> = [
+	'--typography-font-weight-normal: normal;',
+	'--typography-font-weight-bold: bold;',
+	'--typography-font-weight-light: lighter;',
```

Comment:
https://htmldog.com/references/css/properties/font-weight/#:~:text=lighter,inherited%20font%20weight

Reply:
I changed it to numbers.